### DISCUSSION TITLE
adding 2 Similar Sites in 2 different region

### DISCUSSION THREAD

**Discussion by yousefii** [netbox-community/netbox#13070] [Help Wanted!]
Hello friends,

I have two sites located in two different regions that have similar names. When I tried to add the second site, I received an error message stating that the site name already exists.

Is this impossible or have I missed something here?

**Comment by mtinberg** [comment]
Site names need to be unique, including the shortened slug version, much like other database records.  The Region or Site Group hierarchy is maintained outside of that and doesn't affect the uniqueness constraint.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: Iman Yousefi ***@***.***>
Sent: Monday, July 3, 2023 1:36 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] adding 2 Similar Sites in 2 different region (Discussion #13070)


Hello friends,

I have two sites located in two different regions that have similar names. When I tried to add the second site, I received an error message stating that the site name already exists.

Is this impossible or have I missed something here?
